### ISSUE TITLE
Highlight

### ISSUE THREAD

**Issue by gkssjovi** [skattyadz/vscode-quick-scope#2]
It is possible to highlight after I press the "f" like in "neovim"?
Also it is possible to have an underline instead of the highlight similar to the attached image?
![Screenshot 2021-02-12 at 17 52 46](https://user-images.githubusercontent.com/5613459/107790203-309d5180-6d5b-11eb-84da-cf0a89ed682a.png)

**Comment by neildataprophet** [comment]
It would be nice to include this as a setting

**Comment by codeincontext** [comment]
Can I ask which vim plugins you're both using? I don't yet know how to do this, but it might mean listening to events from there

**Comment by codeincontext** [comment]
> Also it is possible to have an underline instead of the highlight similar to the attached image?
> ![Screenshot 2021-02-12 at 17 52 46](https://user-images.githubusercontent.com/5613459/107790203-309d5180-6d5b-11eb-84da-cf0a89ed682a.png)

Yes, I think that is possible. It would mean tweaking the decoration definitions here https://github.com/codeincontext/vscode-quick-scope/blob/master/src/extension.ts#L5-L13

It would be nice to have an option to change it in the extension settings. I'm unlikely to have time to do it myself, but do you have an idea for how that config might look?

**Comment by neildataprophet** [comment]
I think you're right - that was my thought when looking at it (not that I know anything about TypeScript).  I think adding a setting with a default and referencing that. I can take a look when I have a bit more time on my hands.

Just the standard vscodevim plugin.

**Comment by NeonReactionary** [comment]
Sorry, was on the wrong account. You can indeed, change the decorator definitions (https://code.visualstudio.com/api/references/vscode-api#DecorationRenderOptions). I'll just try and find out how to add a reasonable config that's exposed to the user and then submit a pull request.

**Comment by NeonReactionary** [comment]
I may need permission to push a proposed change. Still in progress (unfortunately things have been quite busy, so I haven't had much time to look at it).

**Comment by darianmorat** [comment]
> Sorry, was on the wrong account. You can indeed, change the decorator definitions (https://code.visualstudio.com/api/references/vscode-api#DecorationRenderOptions). I'll just try and find out how to add a reasonable config that's exposed to the user and then submit a pull request.

News about it?...
